src/parser/parser_utils.js

```diff
@@ -338,6 +338,17 @@ function mergeWrapperAdData(unwrappedAd, wrapper) {
       wrapper.blockedAdCategories
     );
   }
+  
+  if (wrapper.creatives?.length) {
+    const wrapperCreativesWithIconsNode = wrapper.creatives.filter(
+      (creative) => creative.icons?.length && !creative.mediaFiles.length
```

Comment:
Maybe adding a small comment on why we're filtering out the creative with mediafile can be usefull here